### PR TITLE
Add create client command

### PR DESCRIPTION
**Pull request by ajouve**
This is a command to create a new client from the command line

```
$ app/console fos:oauth-server:create-client [--redirect-uri] [--grant-type]
```

Example

```
$ app/console fos:oauth-server:create-client --grant-type="refresh_token" --grant-type="client_credentials"
```

### CODE REVIEW

**Comment by ajouve** [comment]
@willdurand Any comment, should I close it ?

**Comment by smilesrg** [comment]
@ajouve can you please resolve merge conflicts?

File: composer.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
+        "symfony/console": "~2.1"
```
Comment (Ener-Getick) [inline composer.json]: wrong composer file

**Comment by Ener-Getick** [comment]
What do you think @willdurand ?

**Comment by oktapodia** [comment]
:+1:

**Comment by renatomefidf** [comment]
I've done a functional test and it works nicely!
One recommendation is to put ~2.1|~3.0 on the composer.json for the symfony/console require, other than that its nice! :)

**Pull request closed without merging**